File: src/main/java/com/hugo/Guess/GameFor.java
package com.hugo.Guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        Random random = new Random();
        Scanner scanner = new Scanner(System.in);
        int secrect = random.nextInt(10) + 1;
        for (int i = 1; i <= 3; i++) {
            int answer;
            System.out.print("(" + i + "/3)Plz input your number : ");
            answer = scanner.nextInt();
            if (answer > secrect) {
                System.out.println("Lower");
            } else if (answer<secrect) {
                    System.out.println("Hight");
            }else{
                System.out.println("Congratulation Answer is :" + answer);
                break;
            }
        }
    }
}

File: src/main/kotlin/com/hugo/Guess/GameFor.kt
package com.hugo.Guess

import java.util.*

fun main(args: Array<String>) {
    val secrect = Random().nextInt(10)+1;
    for (i in 1..4){
        var answer = Scanner(System.`in`).nextInt();
        println("第${i}次輸入:$answer");
        if(answer == secrect){
            println("Congratuation the answer is : $answer");
            break;
        }
    }

}